bindings/wasm/examples/node/private_tangle.js

```diff
@@ -0,0 +1,42 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+const { Client, Config, Document, KeyType, Network } = require('../../node/identity_wasm')
+
+/**
+    This example shows how a DID document can be created on a private tangle.
+    It can be run together with a local hornet node.
+    Refer to https://github.com/iotaledger/one-click-tangle/tree/chrysalis/hornet-private-net
+    for setup instructions.
+**/
+async function createIdentityPrivateTangle() {
+    // This is an arbitrarily defined network name
+    const network = Network.from_name("atoi");
```

Comment:
Standardise example network names to `"custom"`. Even though `"atoi"` is an anagram of `"iota"`, it could be confusing since it's also the name of a well-known function.

```suggestion
    // This is an arbitrarily defined network name
    const network = Network.from_name("custom");
```